doc/modules/ROOT/pages/examples.adoc

```diff
@@ -0,0 +1,46 @@
+= Examples
+James Elliott <james@deepsymmetry.org>; Simon Arnell <simon.arnell@configuredthings.com>; Mattias Holm
+
+The following are some examples of protocol contributed by the community.
+
+## User Datagram protocol
```

Comment:
There are definitely changes I would make to the æsthetics of some of these diagrams if I were representing them as my own, but you have done a nice job setting these up as community contributed, so I am fine with showing them. I would like to make one organizational change, though: Let’s have the `##` level headings identify who contributed them—so in this case “Mattias Holm (@lorrden)”—and if we could make the username a link to his profile page that would be cool, although I can’t remember if you can do that in an AsciiDoc heading. Then the examples themselves would be `###` level headings, which I think will still show in the right-hand page table of contents.

Reply:
Amended in b0c27cd